wsmaster/codenvy-sql-schema/src/main/resources/codenvy-schema/5.0.0/1.1__init.sql

```diff
@@ -0,0 +1,37 @@
+--
```

Comment:
Hi there, i missed that the latest script in che is _2__update_local_links_in_environments.sql_
which means that your's one must be named as _5.0.0-M9/2.1__script_description.sql_, sorry for that.